inst/makeData.R

```diff
@@ -83,5 +83,29 @@ d = load2("../thirdparty/yeast.RData")
 yeast.task = makeMultilabelTask("yeast-example", data = d, target = paste0("label", 1:14))
 save(yeast.task, file = file.path(dn, "yeast.task.RData"), compress = "xz")
 
+# FDA classification
+set.seed(DATASEED)
+gunpoint = load2("../thirdparty/gunpoint.RData")
```

Comment:
How big is this data? Do we absolutely need it? We can't include too much stuff because of CRAN policies.